resources/openssh_server.rb

```diff
@@ -52,7 +56,7 @@ class Chef::Recipe
     cookbook  cookbook
     source    source
     variables config: sshd_config
-    action    template_action
+    action    :create
```

Comment:
Hardcoding `:create` here might break rare cases where the old definition was used with an `:delete` or `:remove` action. I don't think supporting something like `:delete` is necessary, but this might be something we should mention in the CHANGELOG.

Reply:
I was concerned that this would be a blocker for that specific reason.  I considered adding as a `parameter` which could be set within the recipe.  I wasn't sure what the best approach would be.